application-templates/starter/.env

```diff
@@ -1,2 +1,3 @@
 ENABLE_NEW_JSX_TRANSFORM="true"
-FAST_REFRESH="true"
\ No newline at end of file
+ENABLE_I18N_MESSAGE_COMPILATION="true"
```

Comment:
If it's only for CI, maybe we can set the environment variable in the CI step instead of the dotenv file?

Reply:
I moved setting to the build step of CI.

I didn't write the expression to set it true when `app-type` and `template-type` matches 'x', 'y', as that might be unnecessarily complex. 

This essentially has the same effect, loader does not engage when translated files are not there.